src/App.jsx

```diff
@@ -14,33 +15,86 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_FIELD = {
+  ALPHABET: 'alphabet',
+  LENGTH: 'length',
+};
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+const getPreparedGoods = (goods, { sortField, isReverse }) => {
+  const sortedGoods = [...goods].sort((good1, good2) => {
+    switch (sortField) {
+      case SORT_FIELD.ALPHABET:
+        return good1.localeCompare(good2);
+      case SORT_FIELD.LENGTH:
+        return good1.length - good2.length;
+      default:
+        return 0;
+    }
+  });
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  if (isReverse) {
+    return sortedGoods.reverse();
+  }
 
-      <button type="button" className="button is-danger is-light">
-        Reset
-      </button>
-    </div>
+  return sortedGoods;
+};
+
+export const App = () => {
+  const [sortField, setSortField] = useState('');
```

Comment:
The initial state of `sortField` is set to an empty string, which does not correspond to any of the defined sort fields in `SORT_FIELD`. Consider initializing it with a valid sort field value or handling the empty state in the `getPreparedGoods` function.